src/core/chat-api.ts

```diff
@@ -120,6 +146,11 @@ export class ChatApi {
     return this._makeAuthorizedRequest<CreateMessageResponse>(`/chat/v2/rooms/${roomId}/messages`, 'POST', body);
   }
 
+  async updateMessage(roomId: string, serial: string, params: UpdateMessageParams): Promise<UpdateMessageResponse> {
+    const encodedSerial = encodeURIComponent(serial);
```

Comment:
Lucky for us, gorilla mux handles URI encoding/decoding, which is why this wasn't an issue. That being said, its probably good practice to have this in. Should we also encode the roomId too?

Reply:
I think we should encode roomId too actually, will update.